scenarios/adoption/hci.yml

```diff
@@ -55,14 +55,27 @@ libvirt_manager_patch_layout:
         - osp_trunk
 
 networking_mapper_definition_patch:
+  networks:
+    external:
+      network: "172.21.0.0/24"
+      vlan: 44
+      mtu: 1496
   group-templates:
-    osp-controllers:
+    computes:
```

Comment:
which computes is this then in the adoption jobs. I mean, osp_computes is the source computes which then become the destination data plane nodes.

Reply:
these are the computes from the greenfield jobs, we don't use them for adoption (in https://github.com/openstack-k8s-operators/ci-framework/pull/2297/files#diff-60530f309d776a3851cf8d6bfb03bf6c4bb531cc5de3f57e66f5a14021d545f2R27 we set the number of vms of this group to 0). But, since we reuse the base network definition https://github.com/openstack-k8s-operators/ci-framework/blob/main/scenarios/reproducers/networking-definition.yml  we can't undefine anything that is already there. So I need to make sure the ips for unused compute group do not overlap with the ones from osp-compute groups. I'll add a comment clarifying this